save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt

```diff
@@ -103,11 +103,12 @@ class DockerService(private val configProperties: ConfigProperties) {
                 false
             }
 
-    private fun buildBaseImageForExecution(execution: Execution): String {
+    private fun buildBaseImageForExecution(execution: Execution): Pair<String, String> {
         val resourcesPath = File(
             configProperties.testResources.basePath,
             execution.resourcesRootPath,
         )
+        val runCmd = "${if (File(resourcesPath, "run.sh").exists()) "./run.sh || " else ""}./$SAVE_AGENT_EXECUTABLE_NAME"
```

Comment:
Do you think `||` is useful here? This way we won't be able to pass additional arguments to the script, and if we ave checked that the file exists, there is no need for this safe fallback

Reply:
Corrected, misunderstood at first